### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [martinsvoboda/design-system-template.bck#1]
The user @carlsonsantana validate your site template "https://martinsvoboda.github.io/design-system-template/" and found these accessibility errors:
* Page https://martinsvoboda.github.io/design-system-template/:
  * Issue *a5a59c3a-c176-43bd-aa2e-ded9c578b776*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.undefined
    * **Message**: Img element inside a link must not use alt text that duplicates the content of a text link beside it.
    * **Selector**: `html > body > div > a:nth-child(6) > img`
    * **Context**: `<img width="32" height="32" title="" alt="" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAyRpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2V...`
  * Issue *982dd8fb-50ae-4c7e-8a37-cbab56df5b51*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
    * **Message**: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
    * **Selector**: `html > body > div > a:nth-child(6)`
    * **Context**: `<a href="/" class="logo logo-img-2x">
        <img width="32" height...</a>`
  * Issue *bb523ac8-344a-4bcd-a68b-ac82dccbfddc*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H2.EG3
    * **Message**: Img element inside a link must not use alt text that duplicates the content of a text link beside it.
    * **Selector**: `html > body > div > a:nth-child(5) > img`
    * **Context**: `<img width="32" height="32" title="" alt="" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAyRpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2V...`
  * Issue *6a293973-5b3f-4b3a-a20e-5ee0ffa57fe8*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
    * **Message**: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
    * **Selector**: `html > body > div > a:nth-child(5)`
    * **Context**: `<a href="/" class="logo logo-img-1x">
        <img width="32" height...</a>`
  * Issue *c3a82d0e-45ce-4bbd-9824-dbf76afa8a34*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.42:1. Recommendation: change text colour to #6e6e6e.
    * **Selector**: `#suggestions`
    * **Context**: `<div id="suggestions">
        <a href="https://statu...</div>`
  * Issue *c4718cb6-7c4e-456d-a40c-701ffea5d38d*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.53:1. Recommendation: change text colour to #2f71b2.
    * **Selector**: `html > body > div > p:nth-child(3) > a`
    * **Context**: `<a href="https://help.github.com/pages/">read the full documentation</a>`
  * Issue *076ae95e-4459-4524-90ac-31d61f751f64*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head>
    <meta http-equiv="Co...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/287d7816-7188-4edd-b8c9-ae0245acf38b/).
